lib/portlayer/event/collector/vsphere/collector.go

```diff
@@ -169,12 +169,18 @@ func evented(ec *EventCollector, page []types.BaseEvent) {
 			*types.VmMigratedEvent,
 			*types.DrsVmMigratedEvent,
 			*types.VmRelocatedEvent:
-			event = NewVMEvent(page[i])
-		}
 
-		if event != nil {
-			ec.callback(event)
+			// we have an event we need to process
+			ec.callback(NewVMEvent(page[i]))
+		case *types.VmReconfiguredEvent:
+			// reconfigures happen often, so completely ignore for now
+			return
+		default:
+			// log the skipped event
+			e := page[i].GetEvent()
+			log.Debugf("vSphere Event %s for eventID(%d) ignored by the event collector", e.FullFormattedMessage, int(e.Key))
 		}
```

Comment:
I did that in testing and saw no dupes or out of order events, so removed from this code

If we hit an issue due to that then the other logging should provide enough detail...meaning the ability to follow an event from entry to exit